zend_abstract_interface/exceptions/exceptions_common.h

```diff
@@ -0,0 +1,7 @@
+#include <main/php.h>
+
+#if PHP_VERSION_ID < 70100
+
+#define ZEND_STR_MESSAGE "message"
```

Comment:
Now that I think about this a bit more, do you think it would make sense to remove `exceptions_common.h` since each major PHP version will have its own `php<n>/exceptions.h` header?

Reply:
I'm not sure yet. I think we'll see what makes most sense once we actually implement the php 5 and 7 seams. I think it should be changed (or not) at the point of implementing these.